src/pages/Dashboard/components/Calendar/CalendarContainer.tsx

```diff
@@ -1,28 +1,19 @@
 import React, { useState, useEffect } from 'react';
 import moment, { Moment } from 'moment';
 
+import CalendarBody from './CalendarBody';
 import CalendarDailyTasksModal from './CalendarDailyTasksModal';
+import CalendarHeader from './CalendarHeader';
 
-import { TaskAssignation } from '../types';
-
-import {
-  CalendarContainer,
-  CalendarHeader,
-  Body,
-  CalendarHeaderPrev,
-  CalendarHeaderNext,
-  Week,
-  Day,
-  DayDiv,
-  DayDivTask,
-  DayNames,
-} from '../styles/calendar';
+import { TaskAssignation } from '../../../../types';
+
+import { Container } from '../../../../styles/calendar';
```

Comment:
Could you please change to CalendarContainerStyled  as when imported it can be confused with the <Continaer/> component from MUI ?